static/js/btax.js

```diff
@@ -31,7 +31,13 @@ $.each(asset_yr_str, function(index, value){
   });
 });
 
-$('#start-year-select').change(function() {
-  $('#current-year-link').attr('href', '/ccc/?start_year=' + $(this).val());
-  $('#current-year-alert').removeClass('hidden');
+var currentYear = $('#start-year-select').val();
+$('#start-year-select').change(function(e) {
+  $('#current-year-link').attr('href', '/taxbrain/?start_year=' + $(this).val());
```

Comment:
@hdoupe After changing `'/taxbrain/?start_year='` to `'/ccc/?start_year='` on my local, I was able to have the dialogue popping up and working. 

![screen shot 2018-03-16 at 1 05 59 pm](https://user-images.githubusercontent.com/13324931/37534290-cb79a844-291a-11e8-9979-c29240789179.png)

Reply:
Nice, thanks @GoFroggyRun. Everything is looking good now.